### ISSUE TITLE
Missing ModernMetals Equivalents

### ISSUE THREAD

**Issue by ghost** [MinecraftModDevelopment/EndMetals#31]
Boron
Beryllium
Thorium

Applicable to NetherMetals as well. BaseMinerals and UtilityBaseOres also lack support, but I'm not sure if that'd be up to you or them.

**Comment by jriwanek** [comment]
Yup, known, we'll be implementing soon enough, also, you might want to resubmit your PR actually against this repo instead of your own, and against the master-1.12 branch.

**Comment by jriwanek** [comment]
Also, BaseMinerals wouldn't have support, it doesn't need it as said is provided by ModernMetals
